eclair-node/src/main/resources/application.conf

```diff
@@ -2,6 +2,20 @@ eclair {
     enable-kamon = false
 }
 
+kamon.instrumentation.akka {
+  filters {
+    actors {
+      # Decides which actors generate Spans for the messages they process, given that there is already an ongoing trace
+      # in the Context of the processed message (i.e. there is a Sampled Span in the Context).
+      #
```

Comment:
Maybe explain here why we're disabling by default (what's in the PR description)?